Templates/GitlabCIPipeline/.gitlab-ci.yml

```diff
@@ -0,0 +1,550 @@
+variables:
+    #Git strategy is set to "none" but overwrite later to "clone" in the jobs to create release candidate tag and release tag.
+    GIT_STRATEGY: none
+    GIT_CHECKOUT: "false"
+    application: "CBSA"
+    wdEnvironmentFileIntegration: ${WAZI_DEPLOY_SAMPLES}/plum-samples/external-repos/environment-conf/python/EOLEB7-CBSA-Integration.yml
```

Comment:
done

Reply:
`WAZI_DEPLOY_SAMPLES` is a CI/CD variable defined at the group level. It points to the Wazi Deploy samples location on z/OS. This is where the samples are installed and customized (actually in our setup, it is where the Wazi Deploy repository that contains our customization is cloned).

Reply:
We are passing an absolute path, while the Azure Devops pipeline is passing a relative path. So either, we switch to a relative path and specify the absolute folder path in the `wdEnvironmentConfigurations` variable of the `pipelineBackend.config` file, or we just leave it like this.
Also, please consider changing the path in the template since there is no existence of plum-samples anymore (replaced by wazideploy-samples, like in https://github.com/jbyibm/cics-genapp/tree/main/wazideploy-samples).